components/Buttons/src/MDCFloatingButton.h

```diff
@@ -96,6 +118,26 @@ typedef NS_ENUM(NSInteger, MDCFloatingButtonShape) {
 
 - (nullable instancetype)initWithCoder:(nonnull NSCoder *)aDecoder NS_DESIGNATED_INITIALIZER;
 
+- (void)setMinimumSize:(CGSize)minimumSize NS_UNAVAILABLE;
+
+- (void)setMinimumSize:(CGSize)minimumSize forShape:(MDCFloatingButtonShape)shape
+    UI_APPEARANCE_SELECTOR;
```

Comment:
Right now, these will "default" to `MDCFloatingButtonShapeDefault`, but that may not be expected behavior. We may prefer to require each shape value to be fully specified.